field/template.php

```diff
@@ -18,11 +18,11 @@
 	<div class="plugin-prevnext">
 		<a class="prev" href="<?php echo panel()->urls()->index() . '/pages/' . $prev->id() . '/edit'; ?>">
 			<i class="fa fa-chevron-left" aria-hidden="true"></i>
-			<?php echo $prev->title(); ?>
+			<?php echo $prev->title()->excerpt(50); ?>
 		</a>
 
 		<a class="next" href="<?php echo panel()->urls()->index() . '/pages/' . $next->id() . '/edit'; ?>">
-			<?php echo $next->title(); ?>
+			<?php echo $next->title()->excerpt(50); ?>
```

Comment:
@tristantbg Because of Kirby 3 I will not continue to develop this plugin. It's nice to crop the titles if they are too long, but I think CSS overflow ellipse is a better solution for it. Then it's hidden, not removed which is nicer I think, for hovers ect.

Reply:
Mmh yes you're right, it would be more appropriate !